src/styles/_catalog.scss

```
@@ -1,22 +1,34 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(1, $card-width);
```

no need to repeat 1 column
```suggestion
  grid-template-columns: $card-width;
```